src/algorithms/eigsolvers/Arnoldi.m

```diff
@@ -242,10 +291,14 @@
             end
             
             % display
-            if flag
-                warning('eigsolve did not converge.');
-            elseif ~flag && alg.verbosity > Verbosity.warn
-                fprintf('eigsolve converged.\n');
+            if nargout < 3
+                if flag == 1
+                    warning('Found invariant subspace (error = %.5e).\n', conv);
+                elseif flag == 2
+                     warning('Reached maxiter without convergence.\n');
+                elseif ~flag && alg.verbosity > Verbosity.warn
```

Comment:
this is not a warning, and is handled by verbosity, so should execute independent of `nargout`